openpower/package/openpower-pnor/Config.in

```diff
@@ -112,3 +114,11 @@ config BR2_OPENPOWER_TARGETING_ECC_FILENAME
 config BR2_OPENPOWER_PNOR_XZ_ENABLED
         bool "Enable xz compression in PNOR payloads"
         default n
+
+config BR2_OPENBMC_SYSTEM
+        bool "Build for systems using OpenBMC"
+        default n
+
+config BR2_BUILD_VPNOR
+        bool "Create a virtual PNOR image along with the standard image"
+        default y if (BR2_OPENBMC_SYSTEM)
```

Comment:
I'm not sure that every system uses VPNOR. And, @williamspatrick told me to. Maybe he could comment on this?